srcpkgs/opencl-amd/template

```diff
@@ -0,0 +1,65 @@
+# Template file for 'opencl-amd'
+pkgname=opencl-amd
+version=18.50.708488
+revision=1
+create_wrksrc=yes
+hostmakedepends="wget"
+depends="ocl-icd"
+short_desc="OpenCL userspace driver as provided by the amdgpu-pro driver stack"
+maintainer="Adam Buechner <adi51@gmx.de>"
+license="custom"
+homepage="http://www.amd.com"
+archs="x86_64"
+repository=nonfree
+
+_prefix="amdgpu-pro-"
+_postfix="-ubuntu-18.04"
+_major="18.50"
+_minor="708488"
+
+distfiles="https://drivers.amd.com/drivers/linux/${_prefix}${_major}-${_minor}${_postfix}.tar.xz"
+checksum="b7b7bc9796e524e35f1f3ccbd99ba89cea383a0ed5a5357084522da0bdc09d59"
+
+XBPS_FETCH_CMD="wget --referer https://support.amd.com/en-us/kb-articles/Pages/AMDGPU-PRO-Driver-for-Linux-Release-Notes.aspx -N"
+
+post_extract() {
+	mkdir opencl
+	pushd opencl
+	ar x "${wrksrc}/${_prefix}${_major}-${_minor}${_postfix}/opencl-amdgpu-pro-icd_${_major}-${_minor}_amd64.deb"
+	tar xJf data.tar.xz
+	ar x "${wrksrc}/${_prefix}${_major}-${_minor}${_postfix}/opencl-orca-amdgpu-pro-icd_${_major}-${_minor}_amd64.deb"
+	tar xJf data.tar.xz
+	pushd opt/amdgpu-pro/lib/x86_64-linux-gnu
+	sed -i 's|libdrm_amdgpu|libdrm_amdgpo|g' libamdocl-orca64.so
+	popd
+	popd
+
+	mkdir libdrm
+	pushd libdrm
+	ar x "${wrksrc}/${_prefix}${_major}-${_minor}${_postfix}/libdrm-amdgpu-amdgpu1_2.4.95-${_minor}_amd64.deb"
+	tar xJf data.tar.xz
+	pushd opt/amdgpu/lib/x86_64-linux-gnu
+	rm libdrm_amdgpu.so.1
+	mv libdrm_amdgpu.so.1.0.0 libdrm_amdgpo.so.1.0.0
```

Comment:
Why is rename needed?

Reply:
It seems libdrm_amdgpu.so is already provided by another package but the library from the amdgpu-pro package is still needed for OpenCL.